_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+
+There were a range of inventive designs this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto top-mounted conveyor belts which dropped them onto the volcano's surface.
+
+Some of the more notable designs were:
+
+- Team TLC with their consistent, successful catapult.
+- Team SWI with their simple and effective grab-and-go robot which climbed the volcano with a single token.
+- Team CRB with their suction cup and catapult combination which unfortunately had technical issues for much of the competition.
+- Team HAB with their Hawaiian themed robot, topped with a 3D paper pineapple.
+
+The Final
+---------
+
+Student Robotics 2019 concluded with an intense match between PSC, HAM, TLC, and HRS. Team HAM were quick off the mark pushing their tokens into the scoring zones. Team HRS soon followed, violently shoving their tokens across the arena. Team PSC's robot slowly moved 4 of their tokens into a single scoring zone before unfortunately flipping onto its back. Meanwhile, Team TLC quietly moved towards the edge of the volcano and launched a token directly into the Caldera.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube.com/embed/gOeP96OngE0"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+We saw some brilliant robot designs this year, great work by all teams. We hope you enjoyed the competition as much as we did!
+
+If you didn't take part, or you want to enter again next year, the [sign up page](/schools/how_to_enter) for next year's competition will be up later in the year. Get a team together and start talking to your teachers now!
+
+Results
+-------
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
+| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| Online Presence       | [Collyer's](https://www.collyers.ac.uk/) ([Twitter](https://twitter.com/CollyersRobots))
+
+For full details on the awards, please see the [rulebook](/docs/resources/2019/rulebook.pdf).
```

Comment:
I think we should try to include more detail here about the awards. The non-place awards are fairly important -- the competition is about the competitors learning as much as it is about the robots doing well, so the other prizes help encourage that.

At a minimum, please could we include something about why SWI won the committee award? We already have the spiel for that, so I'm hoping that doing so is easy.